docs/guides/taxes.rst

```diff
@@ -19,21 +14,30 @@ There are three ways in which you can configure taxes:
 
 #. Show gross prices to customers in the storefront
 
-   If selected, prices displayed for customers in storefront will be gross. Taxes will be properly calculated at checkout. Changing this setting has no effect on displaying orders placed in the past.
+   If selected, prices displayed for customers in storefront will be gross (only ``Vatlayer``). Taxes will be properly calculated at checkout. Changing this setting has no effect on displaying orders placed in the past.
 
 #. Charge taxes on shipping rates
 
    If selected, standard tax rate will be charged on shipping price.
 
+Vatlayer
+--------
+
+Configuration can be done in dashboard ``Taxes`` tab.
```

Comment:
I would change the order of those three paragraphs. First "Taxes are charged..." then "Vatleyer handles taxes..." and lastly the part about configuration.